mozdownload/scraper.py

```diff
@@ -819,6 +819,9 @@ def binary_regex(self):
 
     def build_filename(self, binary):
         """Return the proposed filename with extension for the binary."""
+        if hasattr(self, 'builds'):
```

Comment:
Localized tinderbox builds will indeed not have a subfolder with a timestamp. Those are all in the same folder, and only the last one is kept. So you better check for `self.locale_build` here.